conf/munin-singularity.config

```diff
@@ -0,0 +1,56 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for Munin Singularity
+ * -------------------------------------------------
+ */
+
+singularity {
+  enabled = true
+  autoMounts = true
+}
+
+params {
+  max_memory = 256.GB
+  max_cpus = 24
+  max_time = 72.h
+  reference_base = '/data0/btb/references/rnafusion'
+  containerPath = "file:///data0/btb/containers"
+  
+  fusioncatcher_ref = "${params.reference_base}/fusioncatcher_ref/human_v90"
+  star_fusion_ref = "${params.reference_base}/star_fusion_ref/GRCh38_v27_CTAT_lib_Feb092018/ctat_genome_lib_build_dir"
+  ericscript_ref = "${params.reference_base}/ericscript_ref/ericscript_db_homosapiens_ensembl84"
+  pizzly_ref = "${params.reference_base}/pizzly_ref"
+}
+
+process {
+  
+  container = "${params.containerPath}/rnafusion_v${params.container_version}.img"
+
+  withName:star_fusion {
+    cpus = { check_max (24, 'cpus')}
+    container = "${params.containerPath}/rnafusion_star-fusion_v${params.container_version}.img"
```

Comment:
For now, I'm ignoring this. I'm still waiting if people would prefer to have tool containers defined by its release number i.e: fusioncatcher_v1.00 or ericscript_v0.5.5. This would mean breaking the current semantic versioning that we have defined.